src/rdb/tests/rdbt.c

```diff
@@ -845,10 +846,11 @@ testm_update_lookup_all(crt_group_t *grp, uint32_t nranks, uint32_t nreplicas,
 		}
 		ldr_rank = test_rank;
 		printf("INFO: replica rank %u is now leader\n", test_rank);
+#endif
 
 		/* Verify data on the rank now that it is leader */
```

Comment:
suggest change the comment to "Verify data on the test_rank"

Reply:
Done.